### PR TITLE
implemented UI for curriculum inventory report rollover.

### LINKED ISSUE #2022: add ability to clone curriculum inventory reports
cloning down to the sequence level.

implement a workflow similar to course cloning.

when cloning a report, the user may specify a different title, description and academic year.

refs #519

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #2022

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit 0d2c35eb9581dc006b47c7f6587636d8203e1ac5

https://deploy-preview-2267--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit b5323b227c12b255c8b570c7e7b8f76016f7a19d

https://deploy-preview-2267--ilios-frontend.netlify.com

File: app/templates/components/curriculum-inventory-report-rollover.hbs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<div class='rollover-form'>
+  <h3 class="title">{{t 'general.curriculumInventoryReportRollover'}}</h3>
+
+  <p class='rollover-summary'>{{t 'general.curriculumInventoryReportRolloverSummary'}}</p>
+
+  <div class='item name'>
+    <label>{{t 'general.name'}}:</label>
+    {{one-way-input
+      value=name
+      update=(pipe (action 'changeName') (action 'addErrorDisplayFor' 'name'))
```
Comment (jrjohnson) [inline app/templates/components/curriculum-inventory-report-rollover.hbs]: Can this be
`update=(pipe (action (mut changeName) (action 'addErrorDisplayFor' 'name'))`? And then remove the changeName action from the components?  I'm not sure how `pipe` would work in that case.

File: app/router.js [hunk 2]
```diff
@@ -66,7 +66,10 @@ Router.map(function() {
     path: 'curriculum-inventory-reports'
   });
   this.route('curriculumInventoryReport', {
-    path: 'curriculum-inventory-reports/:curriculum_inventory_report_id'
+    path: 'curriculum-inventory-reports/:curriculum_inventory_report_id',
+    resetNamespace: true
```
Comment (jrjohnson) [inline app/router.js]: I don't understand `resetNamespace` in this context.  Seems like it results in the same url `/curriculum-inventory-reports/9/rollover` either way. I might be not understanding from the documentation what this actually does.

Comment (stopfstedt) [inline app/router.js]: it's pointless here, good catch. removing it.

File: app/templates/components/curriculum-inventory-report-rollover.hbs (hunk 1 above)
Comment (stopfstedt) [inline app/templates/components/curriculum-inventory-report-rollover.hbs]: i tried `(mut name)` before, and it gave me an error.

**Pull request merged**